### DISCUSSION TITLE
Nearest-Neighbor Filtering for resizing selections and images

### DISCUSSION THREAD

**Discussion by kinker31** [PintaProject/Pinta#758] [Ideas]
The title should be able to say it all, but ideally, I'd also like for it to work like it does in Paint.net, or even how Pinta handles anti-aliasing with it's paintbrush (and maybe another) tool.

**Comment by cameronwhite** [comment]
This was actually implemented recently for the next release, in 6e5545db7941d85921e28ff4cc1c5162173cfe1e, for resizing images
